library/include/hipblaslt-ext-op.h

```diff
@@ -150,6 +150,12 @@ HIPBLASLT_EXPORT hipblasStatus_t hipblasltExtLayerNorm(hipDataType datatype,
  *  input 2-D tensor buffer. can't be nullptr.
  *
  *  @param[in]
+ *  workspace Amax tensor buffer (4k). can't be nullptr.
```

Comment:
Finding anchors and sections in /home/docs/checkouts/readthedocs.org/user_builds/advanced-micro-devices-hipblaslt/checkouts/752/library/include/hipblaslt-ext-op.h:138: error: argument 'workspace' of command @param is not found in the argument list of hipblasltExtAMax(const hipDataType datatype, const hipDataType outDatatype, void *output, void *input, void *workSpace, void *sync, uint32_t m, uint32_t n, hipStream_t stream) (warning treated as error, aborting now)
the documentation...

Reply:
update